packages/jelos/sources/scripts/backuptool

```diff
@@ -4,26 +4,67 @@
 # Copyright (C) 2020-present Shanti Gilbert (https://github.com/shantigilbert)
 # Copyright (C) 2020 Fewtarius (https://github.com/fewtarius)
 
-BACKUPFILE="/storage/roms/backup/BACKUP.zip"
-mkdir -p "/storage/roms/backup"
+# NOTE: to customize your backups, create a backuptool.conf file in
+# /storage/.config like this:
 
-case "${1}" in
-  "restore")
-    systemctl stop emustation
-    unzip -o ${BACKUPFILE} -d /
-    systemctl start emustation
-  ;;
-  "backup")
-    [ -f "${BACKUPFILE}" ] && rm "${BACKUPFILE}"
-    [ -z "$2" ] && systemctl stop emustation
-    zip -9 -r ${BACKUPFILE} \
-    /storage/.cache/bluetooth/* \
-    /storage/.cache/connman* \
-    /storage/.config/system/configs/system.cfg \
-    /storage/.emulationstation/es_*.cfg \
-    /storage/.config/ppsspp/* \
-    /storage/.config/retroarch/* \
+# LOCATIONS=(
+#   /storage/.config/retroarch/*
+#   /storage/roms/savestates/*
+#   /some/other/folder/file.name*
+# )
+
+BACKUPFOLDER="/storage/roms/backup"
+BACKUPFILE="$BACKUPFOLDER/JELOS_BACKUP.zip"
+mkdir -p $BACKUPFOLDER
+
+DEFAULT=(
+    /storage/.config/fancontrol.conf
+    /storage/.config/backuptool.conf
+    /storage/.cache/bluetooth/*
+    /storage/.cache/connman*
+    /storage/.config/system/configs/system.cfg
+    /storage/.config/ppsspp/*
+    /storage/.config/retroarch/*
+    /storage/.config/moonlight/*
+    /storage/.config/game/*
```

Comment:
I added these folders to the default backup

Reply:
Did you test a restore?  We want to make sure they're put back in the same place they're archived from.

Reply:
Good point - I tested the restore on everything except the `game` configs. These are fine except for `hypseus` which won't let you restore as it's read-only.